front/app/containers/Admin/projects/project/phase/phaseParticipationConfig/components/shared/styling.ts

```diff
@@ -8,8 +8,8 @@ import {
 import styled from 'styled-components';
 
 import { Section, SectionField } from 'components/admin/Section';
-import Error from 'components/UI/Error';
-import Warning from 'components/UI/Warning';
+import Error, { Props as ErrorProps } from 'components/UI/Error';
```

Comment:
Resolving some errors that rollup was throwing. Even though we only package the component library, it analyses the whole project now.

Reply:
I think with my changes this is not necessary anymore, I tried removing this and rollup didn't complain. But also doesn't hurt to have it I guess, so we can leave it like this